src/service/performance-impl.js

```diff
@@ -632,13 +633,12 @@ export class Performance {
     const size = Services.viewportForDoc(documentElement).getSize();
     const rect = layoutRectLtwh(0, 0, size.width, size.height);
     return this.resources_.whenFirstPass().then(() => {
-      // TODO(lannka): should avoid this type casting by moving the `getResourcesInRect`
-      // logic here.
-      const resources = /** @type {!./resources-impl.ResourcesImpl} */ (this
-        .resources_);
-      return resources
-        .getResourcesInRect(this.win, rect, /* isInPrerender */ true)
-        .then(resources => Promise.all(resources.map(r => r.loadedOnce())));
+      return whenContentIniLoad(
+        documentElement,
+        this.win,
+        rect,
+        /* isInPrerender */ true
+      );
```

Comment:
note that here is a logic diff:

whenContentIniLoad does not wait for elements like `amp-ad`, `amp-analytics` etc, which I think is the right behavior for performance reporting. Let me know what you think.

Reply:
@dvoytenko

Reply:
Is there a delta from the old behavior in practice? Is it in a way the performance service worked before or some other service?

Reply:
yes, as noted above, the diff is that 
Before: performance-impl's `pc` event waits for all resources in viewport to load
After: performance-impl's `pc` event waits for resources other than 
https://github.com/ampproject/amphtml/blob/f292573c6a7419c9a47f10c5ddd69d0d67e42da6/src/friendly-iframe-embed.js#L61-L66

I think the new behavior is what we actually wanted. Let me know if you think otherwise.

Reply:
Ads and analytics are not pre-render-allowed components. So they were excluded before as well. Are there other deltas?

Reply:
OK, so it means no delta then.